components/centraldashboard/public/components/main-page.pug

```diff
@@ -47,26 +47,30 @@ app-drawer-layout.flex(narrow='{{narrowMode}}',
                         paper-item.menu-item
                             iron-icon(icon='[[item.icon]]')
                             | [[item.text]]
-            template(is='dom-if', if='[[equals(isolationMode, "multi-user")]]')
-                aside.divider
-                a(href$='[[_buildHref("/manage-users", queryParams.*)]]', tabindex='-1')
-                    paper-item.menu-item#contributors Manage Contributors
+            //- template(is='dom-if', if='[[equals(isolationMode, "multi-user")]]')
+            //-     aside.divider
+            //-     a(href$='[[_buildHref("/manage-users", queryParams.*)]]', tabindex='-1')
+            //-         paper-item.menu-item#contributors Manage Contributors
             aside.divider
-            a(href='https://github.com/kubeflow/kubeflow',
-                tabindex='-1', target="_blank")
-                paper-item.menu-item GitHub
-                    iron-icon.external(icon="launch")
-            a(href='https://www.kubeflow.org/docs/about/kubeflow/',
+            //- a(href='https://github.com/kubeflow/kubeflow',
```

Comment:
just removing all kubeflow links as well?

Reply:
Yeah I figured given we were forking the `centraldashboard` we'd try to change our branding problem (where people think the AI Platform is "Kubeflow"). Hoping it'll make for a nicer UX if we can simplify things!

Based on [my understanding](https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)) of the Apache 2.0 license seems like as long as we leave any original copyright notices in place should be fine to modify. Given there are no visible copyright notices in this component seems to me like we're in the clear.

Reply:
ok, cool